pkgs/development/tools/ols/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub, makeWrapper, odin, lib }:
```

Comment:
Prefer `makeBinWrapper` to `makeWrapper`. It prevent issues on Darwin platforms.